tests/api_tests/users/test_views.py

```diff
@@ -191,3 +191,145 @@ def test_get_projects_logged_in_as_different_user(self):
+        self.user_two.save()
+        self.auth_two = (self.user_two.username, 'justapoorboy')
+        self.public_project_user_one = ProjectFactory(title="Public Project User One", is_public=True, creator=self.user_one)
+        self.private_project_user_one = ProjectFactory(title="Private Project User One", is_public=False, creator=self.user_one)
+        self.public_project_user_two = ProjectFactory(title="Public Project User Two", is_public=True, creator=self.user_two)
+        self.private_project_user_two = ProjectFactory(title="Private Project User Two", is_public=False, creator=self.user_two)
+        self.deleted_project_user_one = FolderFactory(title="Deleted Project User One", is_public=False, creator=self.user_one, is_deleted=True)
+        self.folder = FolderFactory()
+        self.deleted_folder = FolderFactory(title="Deleted Folder User One", is_public=False, creator=self.user_one, is_deleted=True)
+        self.dashboard = DashboardFactory()
+
+    def tearDown(self):
+        ApiTestCase.tearDown(self)
+        Node.remove()
+
+    def test_path_Users_User_id_Nodes_user_not_logged_in(self): #~WORK
+        url = "/{}users/{}/nodes/".format(API_BASE, self.user_one._id)
+        res = self.app.get(url)
+        node_json = res.json['data']
+
+        ids = [each['id'] for each in node_json]
+        assert_in(self.public_project_user_one._id, ids)
+        assert_not_in(self.private_project_user_one._id, ids)
+        assert_not_in(self.public_project_user_two._id, ids)
+        assert_not_in(self.private_project_user_two._id, ids)
+        assert_not_in(self.folder._id, ids)
+        assert_not_in(self.deleted_project_user_one._id, ids)
+
+    def test_get_404_path_Users_User_id_user_not_logged_in(self):
+        url = "/{}users/{}/".format(API_BASE, self.private_project_user_two._id)
+        res = self.app.get(url, auth=self.auth_one, expect_errors=True)
+        assert_equal(res.status_code, 404)
+
+    def test_get_404_path_Nodes_Me_user_logged_in(self):
+        url = "/{}nodes/me/".format(API_BASE, self.user_one._id)
+        res = self.app.get(url, auth=self.auth_one, expect_errors=True)
+        assert_equal(res.status_code, 404)
+
+    def test_get_200_path_Users_User_id_user_logged_in(self):
+        url = "/{}users/{}/".format(API_BASE, self.user_one._id)
+        res = self.app.get(url, auth=self.auth_one)
+        assert_equal(res.status_code, 200)
+
+    def test_get_200_path_Users_Me_user_logged_in(self):
+        url = "/{}users/me/".format(API_BASE, self.user_one._id)
+        res = self.app.get(url, auth=self.auth_one)
+        assert_equal(res.status_code, 200)
+
+    def test_path_Users_Me_Nodes_user_logged_in(self):
+        url = "/{}users/me/nodes/".format(API_BASE, self.user_one._id)
+        res = self.app.get(url, auth=self.auth_one)
+        assert_equal(res.status_code, 200)
+
+        node_json = res.json['data']
+
+        ids = [each['id'] for each in node_json]
```

Comment:
No need to change this, just a suggestion - because you're not using `node_json` anywhere else, this line could be refactored to:

```python
ids = [each['id'] for each in res.json['data']]
```

Doing it that way would save a tiny amount of memory, but the real reason to do it is to reduce complexity and make it easier to read.